rpc/src/v1/tests/mocked/eth_pubsub.rs

```diff
@@ -62,28 +62,28 @@ fn should_subscribe_to_new_heads() {
 
 	// Subscribe
 	let request = r#"{"jsonrpc": "2.0", "method": "eth_subscribe", "params": ["newHeads"], "id": 1}"#;
-	let response = r#"{"jsonrpc":"2.0","result":"0x43ca64edf03768e1","id":1}"#;
+	let response = r#"{"jsonrpc":"2.0","result":"0x4333966aca52ad0b","id":1}"#;
```

Comment:
why change this?

Reply:
Did we change PRNG maybe? :/

Reply:
Yes, it looks like [fixed-hash](https://github.com/paritytech/parity-common/blob/master/fixed-hash/src/hash.rs#L510) is using `rand::distributions::Standard::sample` and `OsRng::fill_bytes` is using `getrandom` but I haven't understood the internals...

Reply:
I changed the way, how next_id is generated. As a result the initial value calculated from the test seed was also changed in tests (where deterministic PRNG used).